docs/list_of_apis.jl

```diff
@@ -16,12 +16,9 @@ apis = [
         "Land Model" => "APIs/Land/LandModel.md",
         "Soil Water Parameterizations" =>
             "APIs/Land/SoilWaterParameterizations.md",
-        "Soil Heat Parameterizations" =>
-            "APIs/Land/SoilHeatParameterizations.md",
     ],
     "Common" => [
         "Orientations" => "APIs/Common/Orientations.md",
-        "Spectra" => "APIs/Common/Spectra.md",
```

Comment:
And this?

Reply:
No - I did not .. I think I messed up the rebase